pkg/shell/superuser.jsx

```diff
@@ -30,8 +30,12 @@ function sudo_polish(msg) {
     if (!msg)
         return msg;
 
+    msg = msg.trim();
     msg = msg.replace(/^\[sudo] /, "");
     msg = msg[0].toUpperCase() + msg.slice(1);
+    console.log("P", msg, msg[msg.length-1]);
+    if (msg.length >= 1 && msg[msg.length - 1] == ":")
+        msg = msg.slice(0, -1);
```

Comment:
Nitpicking, but it feels to me that just appending another `.replace(/:$/, "")` to line 34 would be a bit cleaner.

Reply:
Yes, good idea.